sdk/web-pubsub/web-pubsub/CHANGELOG.md

```diff
@@ -4,6 +4,10 @@
 
 ### Features Added
 
+- Support for DefaultAzureCredential
+- Added generateClientToken method to the `HubClient`.
```

Comment:
I think we need `generateClientUrl` containing both client connect path and the token